src/evaluation/Run.jl

```diff
@@ -108,6 +104,23 @@ function run_reactive!(session::ServerSession, notebook::Notebook, old_topology:
 	return new_order
 end
 
+"""
+find (indirectly) deactivated cells and update their status
+"""
+function disable_dependent_cells!(notebook:: Notebook, topology:: NotebookTopology):: Vector{Cell}
+	for cell in notebook.cells
+		cell.depends_on_disabled_cells = false
+	end
+	deactivated = filter(c -> c.running_disabled, notebook.cells)
+	indirectly_deactivated = collect(topological_order(notebook, topology, deactivated))
+	for cell in indirectly_deactivated
+		cell.running = false
+		cell.queued = false
+		cell.depends_on_disabled_cells = true
+	end
+	return indirectly_deactivated
+end
+
```

Comment:
Why are there changes to Run.jl? We already compute the indirectly disabled cells, right?

Reply:
I have to determine the indirectly deactivated cells before saving (in the original implementation it was done afterwards).
In order to avoid double work, run_reactive! gets the list of deactivated cells as (optional) parameter.

See https://github.com/fonsp/Pluto.jl/pull/1209#discussion_r644156923 and https://github.com/fonsp/Pluto.jl/pull/1209#discussion_r644159254